lib/command_line_reporter/row.rb

```diff
@@ -58,12 +58,15 @@ def output
           # | xxxxxxxxxxx |            | (4)
           # +-------------+------------+
           if col.screen_rows[sr].nil?
-            line << ' ' * col.width
+            line << ' ' * col.width << ' '
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.